### PR TITLE
Add tests for BigInt disallowed in LegacyOctalLike and NonOctal literals

### PR DESCRIPTION
**Pull request by leobalter**
Ref https://github.com/tc39/proposal-bigint/issues/208

cc @rkirsling

### CODE REVIEW

File: test/language/literals/bigint/non-octal-like-invalid-09n.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    NonZeroDigit DecimalDigits_opt BigIntLiteralSuffix
+features: [BigInt]
+negative:
+  phase: parse
+  type: SyntaxError
+---*/
+
+$DONOTEVALUATE();
+
+08n;
```
Comment (bakkot) [inline test/language/literals/bigint/non-octal-like-invalid-09n.js]: Given the filename, this should perhaps be `09n`?

File: test/language/literals/bigint/legacy-octal-like-invalid-07n.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+// Copyright (C) 2017 Robin Templeton. All rights reserved.
```
Comment (caiolima) [inline test/language/literals/bigint/legacy-octal-like-invalid-07n.js]: Is this header correct?

Comment (leobalter) [reply test/language/literals/bigint/legacy-octal-like-invalid-07n.js]: I copy and past the files structure usually from the same folder. I changed this file but somehow forgot to save it. Thanks!

**Comment by leobalter** [comment]
Thanks @rkirsling, @bakkot, @devsnek, and @caiolima for the reviews!

**Comment by leobalter** [comment]
I merged it too early. The non octal parts need consensus from TC39. I'll hold on reverting as the meeting starts tomorrow, let's see what should be done to fix this here.

**Comment by bakkot** [comment]
For posterity: the change to the proposal [got consensus](https://github.com/tc39/proposal-bigint/issues/208#issuecomment-514385025), so these tests need not be reverted.

**Pull request merged**